airbyte-integrations/connectors/source-klaviyo/metadata.yaml

```diff
@@ -20,11 +20,21 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  suggestedStreams:
+    streams:
+      - events
+      - campaigns
+      - lists
+      - metrics
+      - flows
   releases:
     breakingChanges:
       1.0.0:
         message: In this release, for 'events' stream changed type of 'event_properties/items/quantity' field from integer to number. Users will need to refresh the source schema and reset events streams after upgrading.
         upgradeDeadline: "2023-11-30"
+      2.0.0:
+        message: In this release, for 'flows' stream changed cursor field from 'created' to 'updated'. Users will need to refresh the source schema and reset events streams after upgrading.
+        upgradeDeadline: "2023-11-30"
```

Comment:
❗ Note that this deadline is today. We should be sure to extend it before releasing this.

Reply:
But this deadline is the end of November